options/docker.go

```diff
@@ -0,0 +1,79 @@
+package options
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+
+	"github.com/docker/docker/client"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/jasper/util"
+	"github.com/pkg/errors"
+)
+
+// Docker encapsulates options related to connecting to a Docker daemon.
+type Docker struct {
+	Host       string
+	Port       int
+	APIVersion string
+	Image      string
+	// Platform refers to the major operating system on which the Docker
+	// container runs.
+	Platform string
```

Comment:
should we expect to add other docker runtime options? or at least provide provisions along those lines.

Reply:
Yeah, I didn't add any options except the bare minimum necessary to verify that Docker containers work. We can add more when we need them.